### ISSUE TITLE
Obsoletion notice protein complex involved in cell adhesion and children  

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#15175]
Dear all,

The proposal has been made to obsolete protein complex involved in cell adhesion and children  #12782
The reason for obsoletion is that this pattern can lead to an infinite explosion of terms. 

There are a few experimental annotations that may need to be rehoused:
12 direct EXP: dictyBase (@PFey), MGI (@ukemi), IntAct (@bmeldal )
'protein complex involved in cell-cell adhesion': 2 direct EXP: dictyBase (@PFey)
'protein complex involved in cell-matrix adhesion': = direct annotations.

One possibility is to annotate to the MF 'cell adhesion mediator activity' 

We are opening a comment period for this proposed obsoletion. We'd like to proceed and obsolete this term on Feb 23, 2018. Unless objections are received by Feb 23, 2018 we will assume that you agree to this change.

Thanks, Pascale

**Comment by bmeldal** [comment]
I can see the rational why things could get inflated along this route. It was one of @dosumis 's restructuring classes.

However, where will 
emilin complex
multimerin complex
tenacin complex
integrin complex
collagen type XVII trimer

go? They were grouped together by GO:0098636 protein complex involved in cell adhesion. I think we called it that instead of making a term for "extracellular matrix complex". Alternatively they will all the tied directly to the root and cannot be found with a grouping term in the complex class.

**Comment by pgaudet** [comment]
Hi @bmeldal 
Thanks for the feedback.  It seems to me protein complexes are mostly linked to the 'protein complelx' root via is_a. The grouping could come via the 'capable of' relation. The creation of a parent class is certainly the exception right now. Perhaps this is a discussion topic for the Protein Complex working group ? 

Thanks, Pascale

**Comment by bmeldal** [comment]
We have lots of functional grouping terms as the "hierarchy" of protein complexes was very flat when I started (and most didn't have logical definitions linking them to their function). 

The idea was to create grouping terms for complexes as well as functions to allow users to search in the complex classes. I'm pretty sure there are still complex classes that still have no logical defs to their function.

And then there are complexes that have no clearly described function in the MF class but can be logically linked by their CC grouping term. One example was subcellular location where that is the common denominator that people would be looking for. Again, they are linked by their part_of relationships but I don't know if all old terms now have the logical defs...

All in all, the ontology is incomplete as we only added these relationships if and when we see the gaps, so CC grouping terms help.
